contrib/inventory/foreman.ini

```diff
@@ -116,6 +127,8 @@ group_patterns = ["{app}-{tier}-{color}",
 group_prefix = foreman_
 # Whether to fetch facts from Foreman and store them on the host
 want_facts = True
+# Whether to create Ansible groups for host collections
+want_hostcollections = False
```

Comment:
After the comment from @agx, I've reverted back to False as the default with commit 635c4fb, to not break existing setups with stand-alone Foreman. 

I don't have time to implement the "auto" logic that @agx suggested and test it with Foreman unfortunately.